config/locals.js

```diff
@@ -40,6 +43,44 @@ function cacheCampaign(phoenixCampaign) {
 }
 
 /**
+ * Create Doing/Completed Mobile Commons Groups to support Mobile Commons broadcasting.
+ * @see https://github.com/DoSomething/gambit/issues/673
+ */
+function createMobileCommonsGroupsForCampaign(campaignModel) {
+  const campaignScope = campaignModel;
+  const prefix = `env=${process.env.NODE_ENV} campaign_id=${campaignModel._id}`;
+
+  // Migrate old campaign models.
```

Comment:
Code was breaking on old DB models that didn't have `mobileCommonsGroups.completed`